bears/general/FilenameBear.py

```diff
@@ -37,8 +37,14 @@ def run(self, filename, file,
         head, tail = os.path.split(filename)
         filename_without_extension, extension = os.path.splitext(tail)
         try:
+            if extension == '.java':
```

Comment:
youch.  this is in a try block designed to catch the KeyError ... nothing else should be in here.